specs/TransportProtocols.md

```diff
@@ -7,24 +7,15 @@ This document describes the protocols used by the three ASP.NET Endpoint Transpo
 A transport is required to have the following attributes:
 
 1. Duplex - Able to send messages from Server to Client and from Client to Server
-1. Frame-based - Messages have fixed length (as opposed to streaming, where data is just pushed throughout the connection)
-1. Frame Metadata - Able to encode one piece of Frame Metadata
-    * `Type` - Either `Binary` or `Text`.
 1. Binary-safe - Able to transmit arbitrary binary data, regardless of content
 1. Text-safe - Able to transmit arbitrary text data, preserving the content. Line-endings must be preserved **but may be converted to a different format**. For example `\r\n` may be converted to `\n`. This is due to quirks in some transports (Server Sent Events). If the exact line-ending needs to be preserved, the data should be sent as a `Binary` message.
 
-Multi-frame messages (where a message is split into multiple frames) may not overlap and all frames **must** have the same `Type` value (`Text` or `Binary`). It is an error to change the `Type` flag mid-message and the endpoint may terminate the connection if that occurs.
-
 The only transport which fully implements the duplex requirement is WebSockets, the others are "half-transports" which implement one end of the duplex connection. They are used in combination to achieve a duplex connection.
 
 Throughout this document, the term `[endpoint-base]` is used to refer to the route assigned to a particular end point. The term `[connection-id]` is used to refer to the connection ID provided by the `OPTIONS [endpoint-base]` request.
 
 **NOTE on errors:** In all error cases, by default, the detailed exception message is **never** provided; a short description string may be provided. However, an application developer may elect to allow detailed exception messages to be emitted, which should only be used in the `Development` environment. Unexpected errors are communicated by HTTP `500 Server Error` status codes or WebSockets `1008 Policy Violation` close frames; in these cases the connection should be considered to be terminated.
 
-## `Close` and `Error` frames
```

Comment:
Decided to kill these since this is being merged into the hub protocol and the individual transport protocols weren't being used. I didn't remove it in the code yet though (we weren't sending these today anyways). I can do that if we agree it's a good idea.